src/plugins/icons.js

```diff
@@ -110,7 +110,7 @@ const customIcons  = {
         },
         tess_links_to_training_materials: {
             type: 'img',
-            icon: 'assets/icons/elixir.jpg'
+            icon: 'assets/icons/elixir.png'
```

Comment:
```suggestion
            icon: 'assets/icons/elixir_tess.png'
```

I'd also suggest renaming the old logo elixir_uk.jpg.

Reply:
@knirirr its in my last commit. tnx